protocol/synthetix/contracts/storage/ScalableMapping.sol

```diff
@@ -45,7 +45,7 @@ library ScalableMapping {
     /**
      * @dev Thrown when attempting to scale a mapping with an amount that is lower than its resolution.
      */
-    error InsufficientMappedAmount(int256 scaleModifier);
```

Comment:
This is probably the only significant change to the contracts of this PR.

I removed this value from the error because it is not very informative beyond the logical reason that triggers it, and it is very hard to calculate and produce a value to check against from the tests without adding a bunch of view functions on ScalableMapping.sol without bloating its interface to a point that might be more confusing for devs using it.